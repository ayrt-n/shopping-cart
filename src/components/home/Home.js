import React from 'react';
import image1 from './images/stussy-home-1.webp';
import image2 from './images/stussy-home-2.webp';
import image3 from './images/stussy-home-3.webp';
import image4 from './images/stussy-home-4.webp';
import image5 from './images/stussy-home-5.webp';
import '../../styles/Home.css';

function Home() {
  return (
    <div className="Home">
      <div className="Home-showcase-container">
        <img src={image1} className="Showcase-item" alt="Man wearing black stussy shirt"/>
        <img src={image2} className="Showcase-item" alt="Man wearing black stussy hoodie"/>
      </div>
        <div class="Home-showcase-container">
          <img src={image3} className="Showcase-item" alt="Stussy denim jacket"/>
          <img src={image4} className="Showcase-item" alt="Man wearing white stussy shirt and brown pants"/>
        </div>
      <div className="Home-showcase-container">
        <img src={image5} className="Showcase-item full-size" alt="Miniature eight ball sitting on countertop"/>
      </div>
    </div>
  );
}

export default Home;
