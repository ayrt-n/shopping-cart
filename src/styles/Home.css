.Home {
  margin: 0 auto;
  position: relative;
  padding-top: 1rem;
  padding-bottom: 2rem;
  top: 3.25rem;
  width: 100%;
}

.Home-showcase-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
}

.Showcase-item {
  width: 33%;
  padding: 0.5rem;
  box-sizing: border-box;
}

.Showcase-item.full-size {
  width: 66%;
  box-sizing: border-box;
}

@media screen and (max-width: 768px) {
  .Showcase-item {
    width: 66%;
  }
}