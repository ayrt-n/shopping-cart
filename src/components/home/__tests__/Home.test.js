import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Home from '../Home';

describe('Home component', () => {
  it('render homepage with images', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
