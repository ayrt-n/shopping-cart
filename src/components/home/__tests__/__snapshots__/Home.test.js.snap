// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component render homepage with images 1`] = `
<div>
  <div
    class="Home"
  >
    <div
      class="Home-showcase-container"
    >
      <img
        alt="Man wearing black stussy shirt"
        class="Showcase-item"
        src="stussy-home-1.webp"
      />
      <img
        alt="Man wearing black stussy hoodie"
        class="Showcase-item"
        src="stussy-home-2.webp"
      />
    </div>
    <div
      class="Home-showcase-container"
    >
      <img
        alt="Stussy denim jacket"
        class="Showcase-item"
        src="stussy-home-3.webp"
      />
      <img
        alt="Man wearing white stussy shirt and brown pants"
        class="Showcase-item"
        src="stussy-home-4.webp"
      />
    </div>
    <div
      class="Home-showcase-container"
    >
      <img
        alt="Miniature eight ball sitting on countertop"
        class="Showcase-item full-size"
        src="stussy-home-5.webp"
      />
    </div>
  </div>
</div>
`;
